README.md

```diff
@@ -276,6 +276,7 @@ Templates in k8s-handle use jinja2 syntax and support all standard filters + som
 * `{{ my_var | b64encode }}` - encode value of my_var to base64
 * `{{ my_var | b64decode }}` - decode value of my_var from base64
 * `{{ my_var | hash_sha256 }}` - encode value of my_var to sha256sum
+* `{{ my_var | to_yaml(flow_style=True, width=99999) }}` - takes dict and render it to yaml(flow_style=True - render in one line, False multiline. width - max line width for rendered yaml lines) 
```

Comment:
AFAIK, safe_dump at least can render dicts, strings (doesn't make much sense though), and sequences: lists and so on.
Probably, that line should be changed to "Tries to render yaml representation of given Python object, if it's supported", or to something similar to reflect it.